src/theme/index.hbs

```diff
@@ -103,6 +103,11 @@
 
         <nav id="sidebar" class="sidebar" aria-label="Table of contents">
             <div class="sidebar-scrollbox">
+                {{#if book_logo }}
+                <div class="sidebar-book-logo">
+                    <img src="{{ book_logo }}">
```

Comment:
Please add an `alt` attribute for e.g., accessibility. Although its content could be generated, because of language differences and because of the principle of least surprise, I propose to add a config field containing the `alt` title.